rule_packages/c/Memory1.json

```diff
@@ -0,0 +1,66 @@
+{
+  "MISRA-C-2012": {
+    "RULE-9-2": {
+      "properties": {
+        "obligation": "required"
+      },
+      "queries": [
+        {
+          "description": "Using braces in initializers of objects and subobjects improves code readability and clarifies intent.",
+          "kind": "problem",
+          "name": "The initializer for an aggregate or union shall be enclosed in braces",
+          "precision": "high",
+          "severity": "warning",
```

Comment:
maybe also have this as recommendation since the cpp rule is

Reply:
Change made.